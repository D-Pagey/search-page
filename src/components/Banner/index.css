.banner-div {
  background-color: #f9e5be;
  padding: 6px;
  text-align: center;
  width: 100%;
}

.banner-link {
  text-decoration: none;
}

.banner-link:hover {
  text-decoration: underline;
}

.banner-link:visited {
  color: black;
}