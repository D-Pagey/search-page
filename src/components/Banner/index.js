import React from 'react';
import styled from 'styled-components';

const BannerWrapper = styled.div`
  background-color: #f9e5be;
  padding: 6px;
  text-align: center;
  width: 100%;
`;

const BannerLink = styled.a.attrs({
  target: '_blank',
  rel: 'noopener noreferrer',
  href: 'http://help.joindrover.com/personal-use-drivers/arranging-delivery-for-your-vehicle',
})`
  color: black;
  text-decoration: none;
  &:hover {
    text-decoration: underline;
  }
  &:visited {
    color: black;
  }
`;

export default function Banner() {
  return (
    <BannerWrapper>
      <BannerLink>
      FREE DELIVERY WITHIN 50 MILES
      </BannerLink>
    </BannerWrapper>
  );
}
