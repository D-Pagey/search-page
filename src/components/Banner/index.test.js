import React from 'react';
import { shallow } from 'enzyme';
import Banner from '.';

it('should render', () => {
  const wrapper = shallow(<Banner />);
  expect(wrapper).toMatchSnapshot();
});
