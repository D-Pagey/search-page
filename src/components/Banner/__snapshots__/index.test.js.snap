// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div
  className="banner-div"
>
  <a
    className="banner-link"
    href="http://help.joindrover.com/personal-use-drivers/arranging-delivery-for-your-vehicle"
    rel="noopener noreferrer"
    target="_blank"
  >
    FREE DELIVERY WITHIN 50 MILES
  </a>
</div>
`;
